django/core/signing.py

```diff
@@ -255,6 +258,11 @@ def unsign_object(self, signed_obj, serializer=JSONSerializer, **kwargs):
 
 
 class TimestampSigner(Signer):
+    def __init__(self, *args, **kwargs):
+        self.expiration_key = kwargs.pop("expiration_key", None)
```

Comment:
Thinking about this in terms of `max_age`, why is this part of the `ctor` and not solely passed to `usign_object`?